SwiftReflector/CustomSwiftCompiler.cs

```diff
@@ -219,13 +219,13 @@ public Version GetCompilerVersion ()
 
 			var modulesInLibraries = SwiftModuleFinder.FindModuleNames (libraryDirectories, CompilerInfo.Target);
 
-			List<ISwiftModuleLocation> locations = SwiftModuleFinder.GatherAllReferencedModules (modulesInLibraries,
-													     includeDirectories, CompilerInfo.Target);
+			var locations = SwiftModuleFinder.GatherAllReferencedModules (modulesInLibraries,
+													     includeDirectories.ToList (), CompilerInfo.Target);
 			string output = "";
 			try {
-				output = Reflect (locations.Select (loc => loc.DirectoryPath), libraryDirectories, pathName, extraArgs, moduleNames);
+				output = Reflect (locations.Select (loc => loc.ParentPath), libraryDirectories, pathName, extraArgs, moduleNames);
 			} finally {
-				locations.DisposeAll ();
+				//locations.DisposeAll ();
```

Comment:
Do you still want this try/finally block?

Reply:
No. No I do not. Thanks!